assets/templates/too_many_oauth_clients.html

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
```

Comment:
Don't forget to add this template to https://github.com/cozy/cozy-stack/blob/master/web/statik/handler.go#L29-L54

Reply:
Ah OK. What's the purpose of this list?